lib/portlayer/storage/vsphere/parent_test.go

```diff
@@ -101,3 +109,49 @@ func TestParentSaveRestore(t *testing.T) {
 		return
 	}
 }
+
+func Session(ctx context.Context, t *testing.T) *session.Session {
```

Comment:
Looks like this and DSsetup below are not used?  Tests above are using datastore.DSsetup, which looks about the same.